src/components/Download/DatasetDetails/ExperimentView/index.js

```diff
@@ -1,41 +1,35 @@
 import { useState } from 'react'
 import { Paragraph } from 'grommet'
 import { useDatasetManager } from 'hooks/useDatasetManager'
+import getFormattedExperiments from 'helpers/getFormattedExperiments'
 import { OrganismFilter } from './OrganismFilter'
 import { ViewBlock } from './ViewBlock'
 import { ViewBlocks } from '../ViewBlocks'
 
-export const ExperimentView = ({
-  dataset: {
-    data: datasetData,
-    id: datasetId,
```

Comment:
I've dropped the renaming of `datasetData` and `datasetId` and instead, used `dataset.data` and `dataset.id` to align with the new approach we decided on.